integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ClouderaManagerEntity.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.it.cloudbreak.newway.entity;
+
+import javax.ws.rs.core.Response;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.cluster.cm.ClouderaManagerV4Request;
+import com.sequenceiq.it.cloudbreak.newway.Prototype;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+@Prototype
+public class ClouderaManagerEntity extends AbstractCloudbreakEntity<ClouderaManagerV4Request, Response, ClouderaManagerEntity> {
```

Comment:
Pls use the new naming convention:
`ClouderaManagerTestDto`
More info: https://jira.cloudera.com/browse/CB-650

Reply:
fixed